prioritization_fees/src/prioritization_fee_data.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{
+    rpc_data::{PrioFeesStats, TxAggregateStats},
+    stats_calculation::calculate_supp_percentiles,
+};
+
+#[derive(Clone, Copy, Debug, Default)]
+pub struct PrioFeesData {
+    pub priority: u64,
+    pub cu_consumed: u64,
+}
+
+impl From<(u64, u64)> for PrioFeesData {
+    fn from(value: (u64, u64)) -> Self {
+        Self {
+            priority: value.0,
+            cu_consumed: value.1,
+        }
+    }
+}
+
+#[derive(Default, Clone)]
+pub struct BlockPrioData {
+    pub tx_prioritization: Vec<PrioFeesData>,
```

Comment:
nit: why do you use the term _prioritization_? the list contains the raw prio fees which have impact on the prioritization but that's not the same